test/utils/combineReducers.spec.js

```diff
@@ -159,16 +159,14 @@ describe('Utils', () => {
               return state + 1
             case 'decrement':
               return state - 1
-            // Never do this in your code:
-            case ActionTypes.INIT:
-              return 0
             default:
-              return undefined
```

Comment:
Okay, I added it [here](https://github.com/rackt/redux/pull/1054/files#diff-91d70a37da68f9dea7fa323102684774R87).